modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -279,3 +280,66 @@ func (suite *TendermintTestSuite) TestCheckHeaderAndUpdateState() {
 		}
 	}
 }
+
+func (suite *TendermintTestSuite) TestPruneConsensusState() {
```

Comment:
I think this test can be written in a cleaner fashion with the new testing package. I believe time increments are 5 seconds. So you should be able to create a client with 10 second trusting period. Update it once and then every subsequent update should prune the oldest consensus state. 

At the very least we should not be testing this function by manually setting the keys

Reply:
Done! Thanks for suggestion!